### ISSUE TITLE
Add function to remove decal in specific area

### ISSUE THREAD

**Issue by Likanin555** [Facepunch/garrysmod-requests#1317]
It would be useful to have a function that would allow you to **remove the decals in the selected area.**

**Comment by ExtReMLapin** [comment]
Perp had something to remove calcs

**Comment by Velkon** [comment]
Yeah the way perp did it is by spamming invisible decals on the location that the towel prop touched, source would remove the old decals in that area, so it would work quite well.

**Comment by ExtReMLapin** [comment]
hackerman

**Comment by mcNuggets1** [comment]
This would actually be really cool, considered I wanted to do an addon about cleaning places.

**Comment by Spanospy** [comment]
This would be a lovely feature, especially for gamemodes like TTT where custom equipment could be made to cleanup a crime scene!

**Comment by Spanospy** [comment]
Very awesome, thanks rubat!!!

Is it easy for this utility to determine the difference between permanent and temporary decals?

Better question - are stuff like bullet holes and blood splatter always temporary unless placed by the map?

**Comment by Spanospy** [comment]
In other words - can we filter for decals to remove by their flags? Should I make this as a new issue?

**Comment by robotboy655** [comment]
Good point, I will add a 4th argument to control removal of permanent decals (defaults to don't remove)

**Comment by DatGuy64** [comment]
@robotboy655 you think this is possible ? https://github.com/Facepunch/garrysmod-requests/issues/2609
